server/api/main.py

```diff
@@ -342,19 +374,23 @@ async def _initiate_logs_collection(start_logs_limit: asyncio.Semaphore):
         run_states.remove(mlrun.common.runtimes.constants.RunStates.aborted)
 
         # list all the runs in the system which we didn't request logs collection for yet
-        runs = await fastapi.concurrency.run_in_threadpool(
+        run_uids = await fastapi.concurrency.run_in_threadpool(
             get_db().list_distinct_runs_uids,
             db_session,
             requested_logs_modes=[False],
-            only_uids=False,
+            only_uids=True,
             states=run_states,
         )
-        if runs:
+        if run_uids:
             logger.debug(
                 "Found runs which require logs collection",
-                runs_uids=[run.get("metadata", {}).get("uid", None) for run in runs],
+                runs_uids=run_uids,
+            )
+            await _start_log_and_update_runs(
+                start_logs_limit=start_logs_limit,
+                db_session=db_session,
+                runs_uids=run_uids,
```

Comment:
maybe align the variable name to `runs_uids`

Reply:
What variable?
There is only `run_uids`

Reply:
`runs_uids=run_uids` should be `runs_uids=runs_uids`